cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,12 @@
+class AddressStepPage {
+    private proceedNextPage: string;
```

Comment:
could you add to the attributes of each page object the type of element we are interacting with? for example in this case, the element is a Button, so the correct names are:
- addressBtn
- confirmAddressButton
- addressButton

choose whatever you want :)